src/tito/builder/main.py

```diff
@@ -454,7 +468,7 @@ def _get_tag_for_version(self, version):
         Can be overridden when custom taggers override counterpart,
         tito.VersionTagger._get_new_tag().
         """
-        return "%s-%s".format(self.project_name, version)
+        return "{}-{}".format(self.project_name, version)
```

Comment:
@dgoodwin sorry, I had not run tests on #261 before you merged it and this was an incorrect mix of print-formatting styles.